Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -795,6 +795,16 @@ def audit_specs
         return if stable_url_minor_version.even?
 
         problem "#{stable.version} is a development release"
+      when %r{^https://github.com/}
+        _, owner, _, _, version = URI.parse(stable.url).path.split("/")
```

Comment:
I'd recommend using 1-2 regexes for this instead which matches `https://github.com/msgpack/msgpack-c/releases/download/cpp-3.2.1/msgpack-3.2.1.tar.gz` or `https://github.com/GoogleCloudPlatform/gcsfuse/archive/v0.29.0.tar.gz` formats.